app/controllers/orga/conferences_controller.rb

```diff
@@ -1,30 +1,14 @@
 class Orga::ConferencesController < Orga::BaseController
-  before_action :find_conference, only: [:show, :edit, :update, :destroy]
+  before_action :find_conference, only: [:show, :destroy]
+
+  def import
+    Conference::Importer.call_with(params[:file])
```

Comment:
> LOL Ruby is all about calling methods, so of course it calls 😉

Haha, you won this round 🤣 

But yeah, `call` is a thing when working with lambdas / Procs:

```ruby
l = ->(cheese) { "#{cheese} is tasty" }
l.call("Camembert") # => "Camembert is tasty"
```

Rack also only expects an arity-1 `call` so that you can either concrete classes/instances reponding to it or just pass a lambda/proc.

I would really prefer `call` instead of adding another name to the mix.